auto_nag/scripts/fuzzing_bisection_without_regressed_by.py

```diff
@@ -46,27 +92,89 @@ def get_bz_params(self, date):
             "f3": "longdesc",
             "o3": "casesubstring",
             "v3": "since this bug contains a bisection range, could you fill (if possible) the regressed_by field",
+            # TODO: Nag in the duplicate target instead, making sure it doesn't already have regressed_by.
+            "f4": "resolution",
+            "o4": "notequals",
+            "v4": "DUPLICATE",
             "emaillongdesc1": "1",
             "emailtype1": "exact",
             "email1": "bugmon@mozilla.com",
         }
 
-    def filter_bugs(self, bugs):
-        # Exclude bugs assigned to nobody.
-        bugs = {
-            bug["id"]: bug
-            for bug in bugs.values()
-            if not utils.is_no_assignee(bug["assigned_to_email"])
-        }
+    def find_regressor_or_needinfo_target(
+        self, bugs: dict[str, dict]
+    ) -> dict[str, dict]:
+        # Needinfo assignee when there is one.
+        for bug in bugs.values():
+            if not utils.is_no_assignee(bug["assigned_to_email"]):
+                bug["needinfo_target"] = {
+                    "mail": bug["assigned_to_email"],
+                    "nickname": bug["assigned_to_nickname"],
+                }
+
+        people = People.get_instance()
 
         # Exclude bugs that do not have a range found by BugMon.
         def comment_handler(bug, bug_id):
```

Comment:
The method grew to be long now, it could be moved to be a private method in the class, this will shift the code more to the left and improve readability.

Reply:
Done in 7e72ad1.